week_3/2018202113HYH/src/tools.h

```diff
@@ -0,0 +1,137 @@
+#ifndef TOOLS_H
+#define TOOLS_H
+
+#include <algorithm>
+#include <vector>
+#include <regex>
+#include <queue>
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <set>
+#include <ctime>
+
+class get
+{
+    public:
+     get()
+    {
+     total_urls=0;
+    }
+
+     get(std::string str,int number = 0)
+    {   
+     total_urls = number;
+     current = str;
+    }
+    void get_path(std::string path)       //改变默认的sterage_path
+    {
+        storage_path = path;
+    }
+    void set_current(std::string str)
+    {
+        current = str;
+    }
+    std::string get_content();
+    int get_num() const         //获得url的总数
+    {
+        return total_urls;
+    }
+    ~get()
+    {
+
+    }
+
+    private:
+
+     int total_urls;
+     std::string current;
+     std::string storage_path = "/home/halcyon/index.html";
+//std::string storage_path;
+
+};
+
+class search;
+class work
```

Comment:
Separate your classes to different files please. Usually one source/header file should have only one class.